app/services/spree_shopify_importer/importers/variant_importer.rb

```diff
@@ -12,20 +12,22 @@ def import!
         data_feed = process_data_feed
 
         if (spree_object = data_feed.spree_object).blank?
-          creator.new(data_feed, @spree_product, @shopify_image).create!
+          creator.new(data_feed, spree_product, shopify_image).create!
         else
-          updater.new(data_feed, spree_object, @spree_product, @shopify_image).update!
+          updater.new(data_feed, spree_object, spree_product, shopify_image).update!
         end
       end
 
       private
 
+      attr_reader :resource, :parent_feed, :spree_product, :shopify_image
```

Comment:
we should talk about this first  and decide about convention

Reply:
It this coherent with the rest of the code ?

Reply:
I'm ready to discuss

Reply:
Thing is if this convention is used in the rest of the code - it is ok to do that here. If it is not - double standards will create a mess.
If we want to use it. Then in separate PR we can do refactor..

Reply:
let me check, and I'll be back with comment

Reply:
Common practice in our codebase is to write `@instance_variable.method_call`, but as for me the way with `attr_reader` is better, and I can do a refactoring when we'll have a time.
I'll try to put a few "pros" of `attr_reader` way below.

Reply:
Convention > personal preference.
I am not opposed to private attr_reader, they are used in other our projects, but in this project not really. We can change that someday. We should not use many different styles/conventions.

Reply:
To be honest I agree with you, I was thinking a bit why I'm writing services in this way and I found that it's just a personal preference or may be just a habit (even from Java time, where all operations with object state carried out using `getters` and `setters`). I'll change a code to follow the convention.
Thanks for discussing.

Reply:
We can change that convention - in many projects we are using attr_readers. Just in separate PRs and for whole namespaces at once. Not a priority right now, though.

Reply:
got it.
Already changed the code.